src/app/modules/browse/browse-controller.js

```diff
@@ -12,6 +13,12 @@
 			title: 'Book Library'
 		});
 	} ])
+    .filter('interpretTag', ['tagService', function(tagService) {
```

Comment:
Filter and the function it calls basically do the same thing...any reason for them not to have the same name? (I think I like getDisplayName better).